### PR TITLE
Vaana 295 initialise client frontend 2

### PR DESCRIPTION
**Pull request by AbdoulayeNDAO121**
Pull request for validating step 2

### CODE REVIEW

File: e2e/protractor.conf.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    require('ts-node').register({
+      project: require('path').join(__dirname, './tsconfig.json')
+    });
+    jasmine.getEnv().addReporter(new SpecReporter({
+      spec: {
+        displayStacktrace: StacktraceOption.PRETTY
+      }
+    }));
+  }
+};
```
Comment (morlombakop) [inline e2e/protractor.conf.js:37]: Please add a new line here

File: package.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "jasmine-core": "~3.6.0",
+    "jasmine-spec-reporter": "~5.0.0",
+    "karma": "~5.2.0",
+    "karma-chrome-launcher": "~3.1.0",
+    "karma-coverage": "~2.0.3",
+    "karma-jasmine": "~4.0.0",
+    "karma-jasmine-html-reporter": "^1.5.0",
+    "protractor": "~7.0.0",
+    "ts-node": "~8.3.0",
+    "tslint": "~6.1.0",
```
Comment (morlombakop) [inline package.json:42]: Please remove `tslint` it is depreciated see https://www.npmjs.com/package/tslint

File: e2e/protractor.conf.js (hunk 1 above)
Comment (AbdoulayeNDAO121) [reply e2e/protractor.conf.js:37]: Hello @morlombakop ,
I don't understand what you want me to do here.

**Comment by morlombakop** [comment]
@thierrymonthe  please do not interfere with PR please. It is strictly  reserved to developers. Unless someone has used your credentials to close this PR. Please kindly change your passwords  if it was not you. cc @AbdoulayeNDAO121

**Pull request closed without merging**